pkg/controller/atlasproject/auditing.go

```diff
@@ -1,38 +1,17 @@
 package atlasproject
 
 import (
-	"encoding/json"
-	"fmt"
 	"reflect"
 
 	"go.mongodb.org/atlas/mongodbatlas"
 
 	"github.com/mongodb/mongodb-atlas-kubernetes/v2/internal/pointer"
 	"github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api"
 	akov2 "github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/api/v1"
-	"github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/controller/customresource"
 	"github.com/mongodb/mongodb-atlas-kubernetes/v2/pkg/controller/workflow"
 )
 
-func ensureAuditing(workflowCtx *workflow.Context, project *akov2.AtlasProject, protected bool) workflow.Result {
-	canReconcile, err := canAuditingReconcile(workflowCtx, protected, project)
-	if err != nil {
-		result := workflow.Terminate(workflow.Internal, fmt.Sprintf("unable to resolve ownership for deletion protection: %s", err))
-		workflowCtx.SetConditionFromResult(api.AuditingReadyType, result)
-
-		return result
-	}
-
-	if !canReconcile {
-		result := workflow.Terminate(
-			workflow.AtlasDeletionProtection,
-			"unable to reconcile Auditing due to deletion protection being enabled. see https://dochub.mongodb.org/core/ako-deletion-protection for further information",
-		)
-		workflowCtx.SetConditionFromResult(api.AuditingReadyType, result)
-
-		return result
-	}
-
+func ensureAuditing(workflowCtx *workflow.Context, project *akov2.AtlasProject) workflow.Result {
```

Comment:
uhm, we already had auditing support! That means the skunkworks PoC may need revisiting

Reply:
auditing in the PoC factors this to be a separate CRD so all is fine here 👍 Auditing currently is a separate field, so the PoC is still of very big value.